### DISCUSSION TITLE
Modern I/O

### DISCUSSION THREAD

**Discussion by cayhorstmann** [java/devjava-content#22] [New Content Ideas]
### Content Details

An article on the more or less recent changes in I/O libraries, showing readers to use modern one-liners instead of loops with `BufferedReader`. An expanded version of https://horstmann.com/unblog/2023-04-09/index.html

### Website Section

Learn -> Tutorials -> Staying Aware of New Features?

**Comment by carimura** [comment]
Moved to issue https://github.com/java/devjava-content/issues/26
